deployments/knack_banner.yaml

```diff
@@ -0,0 +1,57 @@
+###
+### A complete description of a Prefect Deployment for flow 'Knack HR Banner'
+###
+name: HR Knack Banner
+description: Gets updates from HR Banner endpoint and CTM email list and updates Knack
```

Comment:
New update yesterday allows for markdown in the[ description section of your flow!](https://www.loom.com/share/e41ab81d4b954829ab55cd08f207ecb6) 😮